docs/ansible-release-announcement/major-release/announcement-email.md

```diff
@@ -0,0 +1,83 @@
+Release announcement : Ansible Community Package X.0.0
+
+Hello everyone,
+
+We're happy to announce the release of the Ansible X.0.0 package!
+
+Ansible X.0.0 requires latest version of ansible-core a.b and includes a curated set of Ansible collections that provides a vast number of modules and plugins. This is a major release of Ansible.
+
+ansible-core is a required dependency, not contained within the ansible packages. Pip install builds the
+dependency, but it can be built and installed quite independently of the "ansible" distribution.
+
+The "ansible" distribution
+--------------------------
+
+This release is available on PyPI and can be installed with pip:
+
+`$ python3 -m pip install ansible==X.0.0 --user`
+
+The sources for this release can be found here:
+
+Release tarball:
+
+
+SHA256:
+
+
+Wheel package:
+
+
+SHA256:
+
+
+Some important details
+----------------------
+
+Python wheels are now available for both Ansible X and ansible-core a.b resulting in significantly improved installation performance.
```

Comment:
Given that wheels have been available for a bit, can we just drop versions now and say Ansible and ansible-core have wheels?